grails-app/views/report/_dataTableDialog.gsp

```diff
@@ -42,10 +42,11 @@
             "sPaginationType": "two_button",
             "sAjaxSource": "${url}",
             "fnServerData": function ( sSource, aoData, fnCallback ) {
+              const sSourceParsed = sSource.replaceAll("&amp;", "&");
```

Comment:
I am gonna merge this version as a short-term remedy, but I'll create a tech debt ticket to further investigate this issue.